opendihu_examples/isometric_contraction/biceps_muscle/README.md

```diff
@@ -15,6 +15,7 @@ This computes everything in serial and therefore will take a long time. If you c
 ```
 mpirun -n 16 ./muscle_contraction_with_prestretch ../settings_contraction_with_prestretch.py ramp.py --prestretch_force 10.0
 ```
+Instead of 16 ranks you can also (theoretically) choose any other number. Some numbers are not possible because during the paritioning process some paritions might end up empty, which throws an error. 
```

Comment:
```suggestion
`-n` is the number of MPI ranks that will be used. If you choose, e.g., ,`-n 4`, the domain will be partitioned into 4 subdomains. However, not all numbers of ranks are supported, since some partitions might end up empty, which  will throw an error. 
```